### DISCUSSION TITLE
How to import my own model

### DISCUSSION THREAD

**Discussion by JustLmr** [coqui-ai/TTS#4100] [General Q&A]
Hi friends, I want to import my own model, can you help me, my file part is as follows

test
   xtts
      config.json 
      model.index
      model.pth
  main.py

I downloaded a trained model I found on the internet, how can I use my model in my own voice 

note: please tell it like a fool

**Comment by batvanio01** [comment]
Alas, it is not possible, or at least I did not see such an option. There are scripts for additional training, but they only work with the standard model. You can choose yours only maybe if it is a control point type. If your language is on the list of supported ones, you can simply train the standard one with your data. If not, as is the case with me, the only chance is if there is a language close to yours that you can train on yours, as I do.
